### ISSUE TITLE
Description on AutomationProperties page should wrap

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#330]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisite: 
* Settings--> Ease of Access-->Display-->Drag the Display slider  to  200%-->Apply

Repro steps :
1. Launch XAML Controls Gallery.
2. Activate Automation Properties from home page and observe the text under the header "Automation properties"  is not visible.

**Expected behavior**

Information under the header "Automation properties" should be clearly visible in 200% text scaling.

**Actual behavior**

 Information under the header "Automation properties" gets clipped in 200% text scaling

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72850235-863d2280-3c5d-11ea-94b0-609c5be09735.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/24662087)
